cloud/clouds.go

```diff
@@ -55,13 +55,26 @@ const (
 	// OAuth2AuthType is an authentication type using oauth2.
 	OAuth2AuthType AuthType = "oauth2"
 
+	// OAuth2WithCertAuthType is an authentication type using oauth2 and a client certificate
+	// NOTE: This type should never be used in practice as it doesn't exist. We
+	// maintain it here to maintain backwards functionality for Juju in a pre
+	// 2.9 world. Consider using OAuth2AuthType instead.
+	OAuth2WithCertAuthType AuthType = "oauth2withcert"
```

Comment:
Can we group the never use types in a separate const block or even just together in this block

Reply:
Done & Done